### ISSUE TITLE
Stuck at loading call graphs

### ISSUE THREAD

**Issue by bluddy** [inguma/bokken#3]
I'm trying to analyze a binary with the radare2 backend and I get stuck at loading call graph. Any idea what it could be?

**Comment by Jo-Con-El** [comment]
I think the "freeze" is due to xdot running in the same thread as the main GUI.  Were you running the latest revision (with @hteso's changes to xdot.py for gradients and such) or the stable release?

I need that to narrow the scope of this bug and try to alleviate it by e.g. running xdot into a separate thread.

**Comment by Jo-Con-El** [comment]
@bluddy: Any progress?  Can you reproduce it with Bokken 1.7?

**Comment by Jo-Con-El** [comment]
Please reopen if this happens again.  Thanks for reporting!

**Comment by swiftsword94** [comment]
Yeah ... it turns out i'm having the same problem. I don't know if it is exactly the same but the symptoms are the same. Even testing it on a 388 KB file it tends to freeze and not even finish the call graph. Running the same file in IDA returns nearly instantly. Also, this is not just for a single file but for all files. I ran version 1.8 from http://www.bokken.re/ for windows

**Comment by Jo-Con-El** [comment]
Can you provide me a sample of your binary?

**Comment by swiftsword94** [comment]
Unfortunately I shouldn't be using this one as an example but as the problem shows appears with any file I'll use one where it still acts the same. I was just using this to get started on learning assembly so I don't know much about it.
https://drive.google.com/file/d/0B-RkAYpGJIAVWXgwUXotY1d0TUE/view?usp=sharing

**Comment by Jo-Con-El** [comment]
Thanks, I'll take a look at it!

**Comment by HardcoreHacker** [comment]
I've taken a look at CodeBlocks.exe, yes it does seem to hang on the generation of the callgraph. On my Xeon workstation I've let it run for about 45mins, it was still busy, but nothing happened. Now, if I uncheck "Analuze program", it opens quite fast. The problem might be in the way it tries to analyse the program.
